packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.jsx

```diff
@@ -0,0 +1,192 @@
+  const { t } = useTranslation('sso-auth-modal');
+  const show = useMemo(() => !!mode);
+  const authPlugin = shell.getPlugin('auth');
+  const uxPlugin = shell.getPlugin('ux');
+  const { environment } = useApplication();
+  const { user } = environment;
+  const [connectedUser, setConnectedUser] = useState({});
+  const userId = uxPlugin.getUserIdCookie();
+
+  useEffect(() => {
+    document.addEventListener('visibilitychange', () => {
+      setMode(uxPlugin.getSSOAuthModalMode(userId));
+    });
+  }, []);
+
+  useEffect(() => {
+    if (mode === disconnectedToConnected || mode === connectedToOther) {
+      reketInstance
+        .get('/me', {
+          requestType: 'apiv6',
+          params: {
+            target: environment.getRegion(),
+            lang: environment.getUserLocale(),
+          },
+        })
+        .then((currentUser) => {
+          setConnectedUser(currentUser);
+        })
+        .catch((e) => {
+          console.error(e);
+        });
+    }
+  }, [mode]);
+
+  return (
+    <Modal size={size} show={show} backdrop="static" keyboard={false}>
+      <Modal.Body>
+        <div className="sso-auth">
+          <h2 className="oui-modal__title">{t('sso_modal_title')}</h2>
+          <div className="row">
+            {(mode === connectedToDisconnected ||
+              mode === connectedToOther) && (
+              <div
+                className={
+                  mode === connectedToDisconnected ? 'col-md-12' : 'col-md-6'
+                }
+              >
+                <p>{t('sso_modal_user_title')}</p>
+                <div
+                  className={`panel panel-default ${
+                    mode === connectedToOther ? 'panel-disabled' : ''
+                  }`}
+                >
+                  <div className="panel-body">
+                    <div className="sso-modal-account-badge-ctnr">
+                      <div
+                        className="sso-modal-account-badge"
+                        aria-hidden="true"
+                      >
+                        <span className="oui-icon oui-icon-user"></span>
+                      </div>
+                    </div>
+
+                    <div className="sso-modal-account-infos-ctnr">
+                      <div className="sso-modal-account-infos">
+                        <strong>{`${user.firstname} ${user.name}`}</strong>
+                        <br />
+                        <span>{user.email}</span>
+                        <br />
+                        <span>{user.nichandle}</span>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            )}
+            {(mode === disconnectedToConnected ||
+              mode === connectedToOther) && (
+              <div
+                className={
+                  mode === disconnectedToConnected
+                    ? 'col-md-12 span12'
+                    : 'col-md-6 span6'
```

Comment:
IMHO you can safely remove those helpers (they came from Bootstrap 2)

```suggestion
                    ? 'col-md-12'
                    : 'col-md-6'
```

Reply:
removed: done :-)